welcome/ocp-overview.adoc

```diff
@@ -0,0 +1,33 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ocp-overview"]
+= OpenShift Container Platform overview
+
+include::_attributes/common-attributes.adoc[]
+:context: ocp-overview
+toc::[]
+
+{product-title} is a cloud-based Kubernetes container platform. The foundation of {product-title} is based on Kubernetes and therefore shares the same technology. It is designed to allow applications and the data centers that support them to expand from just a few machines and applications to thousands of machines that serve millions of clients.
```

Comment:
REVIEWERS: Though this looks like a new file, the content has been just moved from Getting Started book as part of the reorg effort. Any comments on this file will be addressed in subsequent PRs.